index.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" media="screen" href="./styles.css">
+    </head>
+
+    <body>
+        <!--<section id="portfolio" class="project-section">-->
+            <div>
+                <section class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project1.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h2 class="title">Tonic</h2>
+                        <div class="role">
+                            <div class="role-name">CANOPY</div>
+                            <div class="role-dot"></div>
+                            <div class="role-job">Back End Dev</div>
+                            <div class="role-dot"></div>
+                            <div class="role-year">2015</div>
+                        </div>
+                        <div class="project-details">
+                            <p>
+                                A daily selection of privately personalized
+                                reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <ul class="skills">
+                            <li class="skill set1">html</li>
+                            <li class="skill set2">css</li>
+                            <li class="skill set3">javascript</li>
+                        </ul>
+                        <div class="button-click">
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </section>
+
+                <div class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project2.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h1 class="title">Multi-Post Stories</h1>
```

Comment:
- [ ] In your Headline section, I believe you should have an `h1` tag there. It is HTML best practice that only one `h1` tag should be on a web page. This is because it improves the page ranking on the google search engine. ie. SEO reasons. Kindly replace this `h1` tag with either `h2` or `h3`.